src/DOMTokenList.js

```diff
@@ -23,7 +23,10 @@ module.exports = class DOMTokenList extends Array {
   }
 
   contains(token) {
-    return this.includes(token);
+    for (let i = 0, n = this.length; i < n; i++) {
```

Comment:
I cannot see anything in that link but regardless includes does more than your code so this change won't land in this project